lib/permutations.rb

```diff
@@ -1,4 +1,46 @@
 
-def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
+## Check Permutations
+
+# Write a method which will take two strings as arguments and returns true if one string is a permutation of the other.
+
+# ```
+# permutations?("hello", "ehllo") => true
+# permutations?("pasta", "atsap") => true
+# permutations?("Pizza", "Pasta") => false
+# permutations?("", "") => true
+# ```
+
+
+def permutations?(string1, string2) 
```

Comment:
This is another instance where you've brute-forced the solution.  It works, but a hash would make this much better.  

I encourage you to think a bit further on this.